quirks/change-password-URLs.json

```diff
@@ -86,6 +86,7 @@
     "nytimes.com": "https://myaccount.nytimes.com/seg/profile",
     "orcid.org": "https://orcid.org/account",
     "overleaf.com": "https://www.overleaf.com/user/settings",
+    "app.parkmobile.vo": "https://app.parkmobile.io/account/settings",
```

Comment:
.vo?

Reply:
Whoops! Fixed in 51d44b2c2db7a322e2a7638bac14509162edcf2f